### PR TITLE
Feature/atg 99 Validation forms

### PR DESCRIPTION
**Pull request by IraCherginyshka**

### CODE REVIEW

File: src/app/components/game-login-form/game-login-form.component.ts [hunk 1]
```diff
@@ -60,6 +66,13 @@ export class GameLoginFormComponent implements OnInit {
     });
   }
 
+  forbiddenDateOfBirth(control: FormControl): { [s: string]: boolean } {
```
Comment (oleksandrstarov) [inline src/app/components/game-login-form/game-login-form.component.ts]: can we use more suitable name for function? like validateDateOfBirth?

File: src/app/components/game-login-form/game-login-form.component.html [hunk 2]
```diff
@@ -149,9 +149,10 @@
               class="login__text login__text--warning"
               *ngIf="
                 signUpForm.get('playerInformation.dateOfBirth').invalid &&
-                signUpForm.get('playerInformation.dateOfBirth').touched
+                signUpForm.get('playerInformation.dateOfBirth').touched &&
+                signUpForm.get('playerInformation.dateOfBirth').errors['dateOfBirthIsForbidden']
               "
-              >Please enter your birth date
+              >Please enter your birth date (6+)
```
Comment (oleksandrstarov) [inline src/app/components/game-login-form/game-login-form.component.html]: the 6+ is not really obvious (like you should be over 6 years old to play this game)

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: src/app/components/game-login-form/game-login-form.component.ts (hunk 1 above)
Comment (IraCherginyshka) [reply src/app/components/game-login-form/game-login-form.component.ts]: Changed

File: src/app/components/game-login-form/game-login-form.component.html (hunk 2 above)
Comment (IraCherginyshka) [reply src/app/components/game-login-form/game-login-form.component.html]: Done

**Review by oleksandrstarov — APPROVED** [review]

**Pull request merged**